docs/DeveloperGuide.md

```diff
@@ -295,13 +295,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | user                    | view contacts related to my search query                   | find contacts even when I mistype their name                               |
 | `* *`    | user                    | hide private contact details                               | minimize chance of someone else seeing them by accident                    |
 | `* *`    | user                    | show private contact details                               | view them when I need to                                                   |
+| `* *`    | user                    | have an autocomplete for the names that I am searching     | search faster by names and minimize the chance of an unsuccessful search.  |
```

Comment:
The use of `.` is inconsistent with the previous user stories. Previous user stories do not have `.` to end the sentence. I think it's better to be consistent and stick with whether we want to use `.` or not.